src/psbtv2.ts

```diff
@@ -676,6 +690,63 @@ export class PsbtV2 {
     );
   }
 
+  /**
+   * Other Getters/Setters
+   */
+
+  get nLockTime () {
+    // From BIP0370: The nLockTime field of a transaction is determined by
+    // inspecting the PSBT_GLOBAL_FALLBACK_LOCKTIME and each input's
+    // PSBT_IN_REQUIRED_TIME_LOCKTIME and PSBT_IN_REQUIRED_HEIGHT_LOCKTIME
+    // fields.
+    //
+    // First collect total locks
+    const inputCount = this.PSBT_GLOBAL_INPUT_COUNT;
+    const heightLocks = this.PSBT_IN_REQUIRED_HEIGHT_LOCKTIME;
+    const timeLocks = this.PSBT_IN_REQUIRED_TIME_LOCKTIME;
+    let heights:number[] = [];
+    let times:number[] = [];
+    for (let i = 0; i < this.PSBT_GLOBAL_INPUT_COUNT; i++) {
+      if (heightLocks[i] !== null) {
+        heights.push(heightLocks[i] as number);
+      } 
+      
+      if (timeLocks[i] !== null) {
+        times.push(timeLocks[i] as number);
+      }
+    }
+
+    // From BIP0370: If none of the inputs have a PSBT_IN_REQUIRED_TIME_LOCKTIME
+    // and  PSBT_IN_REQUIRED_HEIGHT_LOCKTIME, then PSBT_GLOBAL_FALLBACK_LOCKTIME
```

Comment:
should this be "or"? so if none of the inputs have time OR height locktime?

Reply:
Probably, but I just copied it from https://github.com/bitcoin/bips/blob/master/bip-0370.mediawiki#determining-lock-time

Reply:
Fixed

Reply:
ha, fair enough